src/PKSim.Core/Services/OverrideScalingMethodSpecification.cs

```diff
@@ -15,15 +18,11 @@ public bool IsSatisfiedBy(ParameterScaling parameterScaling)
          return true;
       }
 
-      public bool IsDefaultFor(ParameterScaling parameterSatifyingSpecification)
+      public bool IsDefaultFor(ParameterScaling parameterScaling)
       {
-         //never the default value
-         return false;
+         return parameterScaling.SourceParameter.IsIndividualMoleculeGlobal();
```

Comment:
It is now the default for those parameters

Reply:
Seems to be wrong for me!
Default method for those parameters should be "Use Source Value", which corresponds to `KeepScalingMethodSpecification` (?) and not to `OverrideScalingMethodSpecification`

Reply:
No you are seeing it backward:
We scale from source to target .From the target point of view, we can either KEEP the value (Use default) or OVERRIDE the value (USe source)

Reply:
This was the old nomenclature. We could probalby rename those classes to match what the UI is showing to avoid confusion.

Reply:
Yep, renaming would be a good idea